README.md

```diff
@@ -11,6 +11,71 @@
 **Interested in helping out?** Take a look at the [contributing
 doc](CONTRIBUTING.md) to find out how.
 
+## Build
+
+The operator is based on the [Operator
+SDK](https://github.com/operator-framework/operator-sdk). In order to build the
+operator, you first need to install the SDK. [Instructions are
+here.](https://github.com/operator-framework/operator-sdk#quick-start)
+
+Once the SDK is installed, Anthill can be built via:
+
+```bash
+$ dep ensure
+
+$ operator-sdk build docker.io/gluster/anthill
+INFO[0001] Building Docker image docker.io/gluster/anthill
+Sending build context to Docker daemon  114.7MB
+Step 1/4 : FROM alpine:3.8
```

Comment:
Do we need to base our dockerimage on alphine ? I think its good to base on CentOS ..  thougts are welome.

Reply:
We should be ok customizing the Dockerfile. This is just what the SDK created. I think we can probably build `FROM scratch`, which would be awesome!

Reply:
I just checked on `FROM scratch`, and that won't work because when you enable testing in the sdk, it injects a shell script into the container. Should be able to move to centos though.

Reply:
Switched to CentOS